configsuite/schema.py

```diff
@@ -57,7 +58,12 @@ def _context_basic_only(elem):
 
 
 def assert_valid_schema(schema):
-    _assert_valid_schema_level(schema)
+    allow_default = MK.Type in schema and isinstance(schema[MK.Type], types.BasicType)
```

Comment:
I guess this should always be `False` since we only allow for defaults inside a `NamedDict`?

Reply:
Yes, it's changed a bit, I'll revert this back slightly